dask_cuda/tests/test_spill.py

```diff
@@ -31,7 +30,8 @@ def device_host_file_size_matches(
     # `dhf.disk` is only available when Worker's `memory_limit != 0`
     if dhf.disk is not None:
         file_path = [
-            os.path.join(dhf.disk.directory, safe_key(k)) for k in dhf.disk.keys()
+            os.path.join(dhf.disk.directory, dhf.disk._safe_key(k))
+            for k in dhf.disk.keys()
```

Comment:
```suggestion
            os.path.join(dhf.disk.directory, fname)
            for fname in dhf.disk.filenames.values()
```

Reply:
Checked this locally and the test pass with that change. Thanks @crusaderky !